README.md

```diff
@@ -37,17 +37,58 @@ For Python projects, you can use the tools in this script with the following
 commands:
 
 ```yaml
-stages:
-  - test
-  - name: deploy
-    if: (branch = master OR tag IS present) AND type != pull_request
+version: ~> 1.0
+
+env:
+  global:
+    # doctr generated secure variable for documentation upload
+    - secure: "<your secure here>"
+    # enable the usage of versions menu which allow versioning of the docs
+    # pages and not only the master branch
+    - DOCTR_VERSIONS_MENU="1"
+    # Dependency files used to build the documentation (space separated)
+    - DOCS_REQUIREMENTS="dev-requirements.txt requirements.txt"
+    # Options to be passed to flake8 for package linting. Usually this is just
+    # the package name but you can enable other flake8 options via this config
+    - PYTHON_LINT_OPTIONS="my_package"
+
+    # The name of the conda package
+    - CONDA_PACKAGE="my_package"
+    # The folder containing the conda recipe (meta.yaml)
+    - CONDA_RECIPE_FOLDER="conda-recipe"
+    # Extra dependencies needed to run the tests which are not included
+    # at the recipe and dev-requirements.txt. E.g. PyQt
```

Comment:
```suggestion
    # in the recipe or CONDA_REQUIREMENTS. E.g. PyQt
```

Reply:
We should also make a point of using the 'test requirements' section in the recipe for this, right?

Reply:
I believe that the tests-requirements section at the recipe will only be used when building the package and running the tests section.

Reply:
Sorry, I accepted the suggestion and github marked it as resolved...

Reply:
What I mean to say is, if we need extra packages to run the tests, that should be captured in the recipe. Just because we make the 'tests' section only do imports (or sometimes nothing at all), doesn't seem to be important.

There could then be a `conda install ... $(get_test_deps_from_recipe recipe-path.yml)`, obviating the need for this env var entirely.

Reply:
(unresolving so we can discuss this a bit more)

Reply:
@klauer I agree and it makes sense!
Looking at the conda documentation I could not find an easy and available way to fetch this dependency list from the recipe.. we may need to roll out our own way.
But I did find something that looks interesting there for the future sometime:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#downstream-tests

Reply:
Sounds good. I think `pcdsutils` can be fairly easily used here: https://github.com/pcdshub/pcdsutils/blob/b713949e0e5a3ebce0805dd3e70f089089596e23/pcdsutils/requirements.py#L91

We could vendor such functions in our ci-helpers to not require installing pcdsutils, perhaps.

Reply:
I will address that in another PR... a new issue was opened to keep track. #35